code/modules/research/xenobiology/crossbreeding/regenerative.dm

```diff
@@ -23,17 +23,42 @@ Regenerative extracts:
 		to_chat(user, span_warning("[src] will not work on the dead!"))
 		return
 	if(H != user)
-		user.visible_message(span_notice("[user] crushes [src] over [H], the milky goo quickly regenerating all of [H.p_their()] injuries!"),
+		user.visible_message(span_notice("[user] crushes [src] over [H], the milky goo quickly regenerating some of [H.p_their()] injuries!"),
 			span_notice("You squeeze [src], and it bursts over [H], the milky goo regenerating [H.p_their()] injuries."))
 	else
-		user.visible_message(span_notice("[user] crushes [src] over [user.p_them()]self, the milky goo quickly regenerating all of [user.p_their()] injuries!"),
+		user.visible_message(span_notice("[user] crushes [src] over [user.p_them()]self, the milky goo quickly regenerating some of [user.p_their()] injuries!"),
 			span_notice("You squeeze [src], and it bursts in your hand, splashing you with milky goo which quickly regenerates your injuries!"))
+
 	core_effect_before(H, user)
-	H.revive(full_heal = TRUE, admin_revive = FALSE)
+	activate_healing(H)
 	core_effect(H, user)
 	playsound(target, 'sound/effects/splat.ogg', 40, TRUE)
 	qdel(src)
 
+/obj/item/slimecross/regenerative/proc/activate_healing(mob/living/target) //No longer full adminheal
```

Comment:
comments like this only make sense for people reading this pr, not people reading this code in the future, properly document this with that in mind https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md